Cargo.toml

```diff
@@ -184,13 +184,14 @@ incremental = true
 debug-assertions = false
 
 [profile.dev]
-opt-level = 2
+opt-level = 1
 lto = "off"
 incremental = true
+debug-assertions = false
 
 [profile.test]
-opt-level = 2
-lto = "thin"
+opt-level = 1
+lto = "off"
 incremental = true
 debug = true
-debug-assertions = true
+debug-assertions = false
```

Comment:
You can introduce profile changes in a separate PR with explanation if you think they're a good idea. In the past changing some of this slowed down CI, and I'm not sure we'd want to turn off debug assertions...

Reply:
oh this is a mistake!